src/main/java/com/gmail/nossr50/config/WorldBlacklist.java

```diff
@@ -3,91 +3,55 @@
 import com.gmail.nossr50.mcMMO;
 import org.bukkit.World;
 
-import java.io.*;
-import java.util.ArrayList;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
 
 /**
  * Blacklist certain features in certain worlds
  */
 public class WorldBlacklist {
-    private static ArrayList<String> blacklist;
+    private static final Set<String> blacklist = new HashSet<>();
```

Comment:
Why is this even static? It's populated by the `WorldBlacklist` instance.
So to me a Singleton pattern would be more sensible here.

If we want to use it in the static methods below, we could just use the singleton instance like
`instance.blacklist.contains(...)`